### PR TITLE
Open Trivia DB (Independent Publisher)

### PR DESCRIPTION
**Pull request by Kiveshan**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Kiveshan** [comment]
> @Kiveshan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement
> ### Contribution License Agreement
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
> 1. **Definitions**.
>    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>    that is delivered by You to Microsoft under this Agreement.
>    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>    Project, including but not limited to communication on electronic mailing lists, source code control
>    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>    discussing and improving that Project, but excluding communication that is conspicuously marked or
>    otherwise designated in writing by You as “Not a Submission.”
>    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>    associated comments and documentation.
> 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>    described in Section 4 below), Submit to any Project.
> 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>    Should You wish to Submit materials that are not Your original work, You may Submit them separately
>    to the Project if You (a) retain all copyright and license information that was in the materials as You
>    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>    containing materials of a third party:” followed by the names of the third party and any licenses or other
>    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>    guidelines concerning Submissions.
> 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>    Submission is made in the course of Your work for an employer or Your employer has intellectual
>    property rights in Your Submission by contract or applicable law, You must secure permission from Your
>    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>    Agreement will refer to You and the employer collectively. If You change employers in the future and
>    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>    and secure permission from the new employer before Submitting those Submissions.
> 5. **Licenses**.
> 
> * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>   parties.
> * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>   Your patent claims that are necessarily infringed by the Submission or the combination of the
>   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>   import or otherwise dispose of the Submission alone or with the Project.
> * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>   granted by implication, exhaustion, estoppel or otherwise.
> 
> 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>    have disclosed under Section 3). You represent that You have secured permission from Your employer to
>    make the Submission in cases where Your Submission is made in the course of Your work for Your
>    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>    You later become aware that would make Your representations in this Agreement inaccurate in any
>    respect.
> 8. **Information about Submissions**. You agree that contributions to Projects and information about
>    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>    information that You submit with Your Submission.
> 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>    defenses of lack of personal jurisdiction and forum non-conveniens.
> 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>     supersedes any and all prior agreements, understandings or communications, written or oral, between
>     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

@microsoft-github-policy-service agree [company="default"]

**Comment by Kiveshan** [comment]
> > @Kiveshan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> > ```
> > @microsoft-github-policy-service agree [company="{your company}"]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > > Options:
> > > 
> > > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > 
> > Contributor License Agreement
> > ### Contribution License Agreement
> > This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> > 
> > 1. **Definitions**.
> >    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
> >    that is delivered by You to Microsoft under this Agreement.
> >    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
> >    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
> >    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
> >    Project, including but not limited to communication on electronic mailing lists, source code control
> >    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
> >    discussing and improving that Project, but excluding communication that is conspicuously marked or
> >    otherwise designated in writing by You as “Not a Submission.”
> >    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
> >    associated comments and documentation.
> > 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
> >    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
> >    described in Section 4 below), Submit to any Project.
> > 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
> >    Should You wish to Submit materials that are not Your original work, You may Submit them separately
> >    to the Project if You (a) retain all copyright and license information that was in the materials as You
> >    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
> >    containing materials of a third party:” followed by the names of the third party and any licenses or other
> >    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
> >    guidelines concerning Submissions.
> > 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
> >    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
> >    Submission is made in the course of Your work for an employer or Your employer has intellectual
> >    property rights in Your Submission by contract or applicable law, You must secure permission from Your
> >    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
> >    Agreement will refer to You and the employer collectively. If You change employers in the future and
> >    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
> >    and secure permission from the new employer before Submitting those Submissions.
> > 5. **Licenses**.
> > 
> > 
> > * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
> >   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
> >   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
> >   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
> >   parties.
> > * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
> >   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
> >   Your patent claims that are necessarily infringed by the Submission or the combination of the
> >   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
> >   import or otherwise dispose of the Submission alone or with the Project.
> > * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
> >   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
> >   granted by implication, exhaustion, estoppel or otherwise.
> > 
> > 
> > 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
> >    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
> >    have disclosed under Section 3). You represent that You have secured permission from Your employer to
> >    make the Submission in cases where Your Submission is made in the course of Your work for Your
> >    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
> >    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
> >    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
> >    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
> >    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
> >    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
> >    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
> >    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> > 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
> >    You later become aware that would make Your representations in this Agreement inaccurate in any
> >    respect.
> > 8. **Information about Submissions**. You agree that contributions to Projects and information about
> >    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
> >    information that You submit with Your Submission.
> > 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
> >    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
> >    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
> >    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
> >    defenses of lack of personal jurisdiction and forum non-conveniens.
> > 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
> >     supersedes any and all prior agreements, understandings or communications, written or oral, between
> >     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.
> 
> @microsoft-github-policy-service agree [company=default - no company specified]

**Comment by Kiveshan** [comment]
> @Kiveshan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="default"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

**Comment by Kiveshan** [comment]
> > @Kiveshan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> > ```
> > @microsoft-github-policy-service agree [company=""]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > > Options:
> > > 
> > > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > 
> > Contributor License Agreement

**Comment by Kiveshan** [comment]
Eg3
Eg
Eg2

**Comment by Kiveshan** [comment]
> @Kiveshan please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="no company specified"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree [company="no company specified"]

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree [company="no company specified"]

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree [company=no company specified]

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="no company specified

**Comment by Kiveshan** [comment]
@microsoft-github-policy-service agree company="no company specified"

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:4]: Hi @Kiveshan, Please change this to your connector name.

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:5]: @Kiveshan, connector description should end with a period. And please make sure that all files are free of grammatical errors. here you have "to allows" and I think you meant to say, "to allow".

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",
+    "description": "This is a connector to the Open Trivia Database to allows access to it's fun trivia questions",
+    "version": "1.0",
+     "contact": {
+      "name": "Kiveshan Naidoo, Rishay Sunder",
+      "email": "kiveshannaidoo9@gmail.com"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:9]: @Kiveshan, please add a "url", you can see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:23]: @Kiveshan,, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+        "summary": "Get Categories",
+        "description": "Get the categories along with their IDs'",
+        "operationId": "GetCategories",
+        "parameters": []
+      }
+    },
+    "/api.php": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:37]: @Kiveshan, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+            "description": "Type",
+            "x-ms-summary": "Type of question"
+          }
+        ]
+      }
+    },
+    "/api_count.php": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:119]: @Kiveshan, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix all responses in the swagger file

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+            "description": "Category",
+            "x-ms-summary": "Category"
+          }
+        ]
+      }
+    },
+    "/api_count_global.php": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:168]: @Kiveshan, default responses are not allowed. Please define a proper response. please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix all responses in the swagger file

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:25]: @Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+        "operationId": "GetCategories",
+        "parameters": []
+      }
+    },
+    "/api.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:39]: @Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+          }
+        ]
+      }
+    },
+    "/api_count.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:121]: @Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 11]
```diff
@@ -0,0 +1,201 @@
...
+          }
+        ]
+      }
+    },
+    "/api_count_global.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:170]: @Kiveshan empty Schema is not allowed, please define a proper schema or you can delete the empty schema.

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 12]
```diff
@@ -0,0 +1,201 @@
...
+    "/api_category.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Categories",
+        "description": "Get the categories along with their IDs'",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:29]: @Kiveshan all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json [hunk 13]
```diff
@@ -0,0 +1,201 @@
...
+    "/api.php": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Question",
+        "description": "Gets a certain amount of questions",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiDefinition.swagger.json:43]: @Kiveshan all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/README.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+
+## Prerequisites
+
+None
+
+## Supported Operations
+
+### Get a certain number of trivia questions, based on a criteria
+
+Get a certain amount of questions from the database based on categories such as diffuculty, Question Type and Category. Leaving
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/README.md:16]: @Kiveshan please make sure that all files are free of grammatical errors. here you have "diffuculty" and I think you meant to say "difficulty".

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json [hunk 15]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json:4]: @Kiveshan all independent publisher must use "#da3b01" for their icon brand color. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json)

File: independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json [hunk 16]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Kiveshan Naidoo"
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/OpenTriviaDB Proposal- OTDB (Independent Publisher)/apiProperties.json:6]: @Kiveshan please add 
    "stackOwner": "Name of company or organization that connector is connecting to."

**Review by Amjed-Ayoub — DISMISSED** [review]

**Comment by Kiveshan** [comment]
@Amjed-Ayoub I apologize, I deleted the previous files by accident (thought I was just messing with my own fork and I did not realise the changes would appear here). However, the new files uploaded have the changes you asked for. I also managed to change from static lists to dynamic ones. Please let me know if anything else needs changing

**Comment by Amjed-Ayoub** [comment]
Hello @Kiveshan,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
@Kiveshan May I ask you please to change the folder name and remove the word Proposal from it.
thank you so much

**Comment by Kiveshan** [comment]
> @Kiveshan May I ask you please to change the folder name and remove the word Proposal from it. thank you so much

@Amjed-Ayoub  Change has been made

**Comment by Kiveshan** [comment]
> @Kiveshan Thank you so much for your hard work on this. Apologies for the multiple requests. i wanted to ask you again to remove the (independent publisher) from the folder name as well. It should have your connector name only. Thank you so much agian

@Amjed-Ayoub Changes made. No worries. Do let me know if anything thing else needs fixing

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @Kiveshan,
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @Kiveshan,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@Kiveshan Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**